### PR TITLE
Add pr template

### PR DESCRIPTION
**Pull request by jvoll**
## Changes
- added PR template

## How to test-drive this PR
- read it!

## TODOS:
- [ ] ~~Update tutorial documentation in the `dev` folder to match the modified tutorial.~~
- [ ] ~~Preview the docs to make sure your changes look good (see [here](https://github.com/mobify/astro#documentation) for instructions on how to preview).~~
- [ ] ~~Change works in both Android and iOS.~~
- [ ] +1 from an engineer on the Astro team.

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+(brief description / PR title)
+
+** Don't forget to label this PR as either: Open for visibility or Ready for review! **
+
+JIRA: **(link to JIRA ticket)**
```
Comment (jeremywiebe) [inline PULL_REQUEST_TEMPLATE.md]: Let's add a "Reviewers: **At least one core Astro engineer**" line here too.

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+## Changes
+- (change1)
+
+## How to test-drive this PR
+- (necessary config changes)
+- (necessary corresponding PRs)
+- (how to access the new / changed functionality -- fixtures, URLs)
+
+## TODOS:
+- [ ] Update tutorial documentation in the `dev` folder to match the modified tutorial.
```
Comment (jeremywiebe) [inline PULL_REQUEST_TEMPLATE.md]: Should we specify that these are the docs in the `astro` repo? Or is it obvious given that this repo doesn't have a `docs` folder?

Comment (jvoll) [inline PULL_REQUEST_TEMPLATE.md]: totally thought I did that. good catch!

**Comment by jeremywiebe** [comment]
:+1:

**Pull request merged**